api/src/Entity/ScheduleEntry.php

```diff
@@ -62,18 +120,25 @@ public function getDay(): Day {
         })->first();
     }
 
+    #[ApiProperty(readable: false)]
     public function getNumberingStyle(): ?string {
         return $this->activity?->category?->numberingStyle;
     }
 
-    public function getDuration(): \DateInterval {
```

Comment:
Wieso fällt getDuration weg?

Reply:
Es wurde nie verwendet ausser in einem Test, und ich habe versucht möglichst keinen toten Code zu migrieren. Hast du einen konkreten Use Case wo wir das bereits morgen brauchen?

Reply:
Das hatten wir im Frontend angezeigt.

Reply:
Es war aber nicht mehr im hydrator drin? Oder habe ich falsch geschaut?